### ISSUE TITLE
Pinning without managed key: is it possible?

### ISSUE THREAD

**Issue by t184256** [cachix/cachix#544]
I like pins, but I don't like the idea of uploading my private key and authenticating with tokens.

Can I use pins with CACHIX_SIGNING_KEY instead of CACHIX_AUTH_TOKEN? Currently it's insisting on auth tokens.

**Comment by domenkozar** [comment]
You will still need a personal auth token, even if you use the signing key.

**Comment by t184256** [comment]
Yeah, I can pin manually with the non-repo-specific token, but that one I don't want to add to GitHub CI because it's wider than just one cache, isn't it? Am I confusing user auth token with cache auth tokens?

**Comment by domenkozar** [comment]
We could allow generating cache tokens even if you have the signing key. That would help :) Ideally we'd just use OIDC from github.
